testing/cypress/e2e/tests/footer-id-function.cy.js

```diff
@@ -0,0 +1,110 @@
+/// <reference types="cypress" />
+
+
+describe('Footer Id Function', () => {
+  
+  // before('create test footer link', () => {
```

Comment:
I had it set up to create the link in a before hook and then delete it in an after hook... but thought it might be better to have it all in the one test to check the function. 

how it is set up now had it in the function testing step.. let me know if you think this is good or if having it in the before and after hook would be better. 

I felt it would be best in the test since those steps are just relevant for that function test.

Reply:
Remove the commented test.